3_for.py

```diff
@@ -16,12 +16,25 @@
 * Посчитать и вывести среднее количество продаж всех товаров
 """
 
-def main():
-    """
-    Эта функция вызывается автоматически при запуске скрипта в консоли
-    В ней надо заменить pass на ваш код
-    """
-    pass
-    
+def main(sales_figures):
+    total_sales = average_sales = 0
```

Comment:
Более привычный вариант
```suggestion
    total_sales, average_sales = 0, 0
```